src/main/flight/mixer.c

```diff
@@ -314,10 +307,28 @@ static void applyFlipOverAfterCrashModeToMotors(void)
             }
         }
 
+        const float flipRateLimit = mixerConfig()->crashflip_rate * 10.0f; // eg 35 = no power by 350 deg/s
+        float flipRateAttenuator = 1.0f;
+        if (flipRateLimit > 0) {
+            // get the gyro rate for the fastest controlled axis
+            float gyroRate = 0.0f;
+            if (signYaw != 0) {
+                // only yaw is used
+                gyroRate = fabsf(gyro.gyroADCf[FD_YAW]);
```

Comment:
What is this supposed to do? Yaw is not used in turtle mode. You cannot flip the quad using yaw. Or am I missing something?